src/main/java/com/github/alexmodguy/alexscaves/server/misc/ACTagRegistry.java

```diff
@@ -93,6 +93,8 @@ public class ACTagRegistry {
     public static final TagKey<Structure> GINGERBREAD_MEN_WANDER_THROUGH = registerStructureTag("gingerbread_men_wander_through");
     public static final TagKey<DamageType> DEEP_ONE_IGNORES = registerDamageTypeTag("deep_one_ignores");
     public static final TagKey<Fluid> DOES_NOT_FLOW_INTO_WATERLOGGABLE_BLOCKS = registerFluidTag("does_not_flow_into_waterloggable_blocks");
+    public static final TagKey<Item> HAZMAT_PROT = registerItemTag("hazmat_prot");
```

Comment:
Considering hazmat is just any hazardous material and you're only using it for radiation, this should have a more specific name than hazmat.
Also I think these tags should have full names such as acid_protective_armor.